src/components/Settings/integrations/Organization/ConnectOrganization.tsx

```diff
@@ -171,7 +199,11 @@ export const ConnectOrganization: React.FC<ConnectOrganizationProps> = ({
               }
               fullWidth
               renderInput={(params) => (
-                <TextField {...params} label={t('Organization')} />
+                <TextField
+                  {...params}
+                  label={t('Organization')}
+                  inputRef={firstFocusRef}
```

Comment:
```suggestion
                  inputRef={(ref) => {
                    ref.focus();
                  }}
```

Reply:
Amazing! I wrapped the function in a callback and adding it to inputRef.